flask_bootstrap/templates/bootstrap/form.html

```diff
@@ -246,13 +246,13 @@
     </form>
 {%- endmacro %}
 
-{% macro render_form_row(fields, row_class='form-row', col_class_default='col', col_map={}) %}
+{% macro render_form_row(fields, row_class='form-row', col_class_default='col', col_map={}, button_map={}) %}
 {#
 fields: an iterable of fields to render in a row.
 row_class: Class to apply to the div intended to represent the row, like
   'form-row' or 'row'.
 col_class_default: Default col class to apply if nothing more specific is said about a field.
-col_class_dict: A dictionary which keys are fields.name and which values are the col class to apply to the field.
+col_map: A dictionary which keys are fields.name and which values are the col class to apply to the field.
 
```

Comment:
a line to describe "button_map" should be added to here.

Reply:
See https://github.com/z25/bootstrap-flask/commit/a1799e071a41eff0ff3ef7a6e5b7bea2cbba67d5